desc/magnetic_fields.py

```diff
@@ -374,6 +374,141 @@ def save_BNORM_file(
         np.savetxt(f"{fname}", data, fmt="%d %d %1.12e")
         return None
 
+    def save_mgrid(
+        self,
+        path,
+        Rmin,
+        Rmax,
+        Zmin,
+        Zmax,
+        nR=101,
+        nZ=101,
+        nphi=90,
+    ):
+        """Save the magnetic field to an mgrid NetCDF file.
```

Comment:
The regular mgrid format allows you to have multiple coil groups with independent currents. We don't necessarily have to support that but we should mention that it only saves as one group with a single current.

Also i think the regular mgrid files also have a field for whether the currents the field was computed with are "raw" or "scaled"? I forget which is which bit might need to add that for codes that expect it

Reply:
Separating the coils into groups or all in one group is arbitrary if you only need the total magnetic field at the end of the day, so I put them all into a single group for simplicity.

Reply:
And the "raw" vs "scaled" is already in here -- it's the `mgrid_mode` variable and I have it hard-coded to "raw" (but the coil current `raw_coil_cur` is also hard-coded to 1)